INSTALL.sh

```diff
@@ -1,8 +0,0 @@
-rm -rf ~/soulmate
-sudo easy_install pip
-git clone --recursive --recurse-submodules https://github.com/Soulmate-Lights/soulmate-core.git ~/soulmate
-python -m pip install --user -r ~/soulmate/esp-idf/requirements.txt
-ln -s ~/soulmate/bin/soulmate /usr/local/bin/soulmate
-
-# If you need to install the drivers, open this app:
-# open ~/soulmate/drivers/Install\ CP210x\ VCP\ Driver.app
```

Comment:
RIP! This tool was kinda cool, it copied your code into the builder directory so your Soulmate "project" could just have sketch.ino and wouldn't have to have all the makefile stuff.

But in retrospect it probably should have been doing something smart with command-line arguments to `make` instead.